static/bidder-params/adtarget.json

```diff
@@ -14,7 +14,10 @@
       "description": "An ID which identifies the site selling the impression"
     },
     "aid": {
-      "type": "integer",
+      "type": [
+        "integer",
+        "string"
+      ],
```

Comment:
requesting to add json test where aid is string

Reply:
adapters/adtarget/params_test.go:45 adapters/adtarget/params_test.go:61